source/Mandrill-Core.package/MandrillClient.class/instance/sendRequest..st

```diff
@@ -2,18 +2,34 @@ as yet unclassified
 sendRequest: aMandrillRequest
 	| response json |
 	self prepareRequest: aMandrillRequest.
-	response := ZnClient new
+	response := [ ZnClient new
 		url:
-				(String
+			(String
+				streamContents: [ :stream | 
+					stream
+						nextPutAll: self class baseUrl;
+						nextPutAll: aMandrillRequest commandUrl ]);
+		entity:
+			(ZnEntity
+				with: (NeoJSONWriter toStringPretty: aMandrillRequest)
+				type: ((ZnMimeType main: 'application' sub: 'json') charSet: 'utf-8'));
+		post ]
+		on: ConnectionTimedOut
```

Comment:
I'm not sure ConnectionTimedOut is a good choice. Is NetworkError not the better one?

Reply:
Yes. ConnectionTimedOut was the one I experimented. But indeed NetworkError would be more exhaustive and still suitable